serge/configs/caps_ts_config.serge

```diff
@@ -189,7 +189,7 @@ jobs
             # plugin for more information.
             data
             {
-                path_matches \/(about|title|description)$
+                path_matches \/(content)$
```

Comment:
Can you change this to:
```path_matches \/(about|title|description|content)$```

Reply:
Ah that makes sense, originally I was just doing the google doc words but those other parts would be useful to translate too.

Reply:
Also if that's used for wordpress that's important to keep